packages/utilities-react/src/StepFlow/StepContext.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useContext, createContext, ReactNode, useState } from 'react';
+import { StepContextType } from './types';
+
+const StepContext = createContext<StepContextType | undefined>(undefined);
+
+interface StepProviderProps {
+  children?: ReactNode;
+}
+
+export const StepProvider: React.FC<StepProviderProps> = ({ children }) => {
```

Comment:
Suggestion not to use `FC`. See https://github.com/carbon-design-system/carbon/pull/20396 and the pull request that's linked in the descriptionn

Reply:
Sure, I've update both places to not use it anymore